Source/Core/writeTextToCanvas.js

```diff
@@ -72,32 +157,28 @@ function writeTextToCanvas(text, options) {
   context2D.lineWidth = strokeWidth;
   context2D[imageSmoothingEnabledName] = false;
 
-  // textBaseline needs to be set before the measureText call. It won't work otherwise.
-  // It's magic.
-  context2D.textBaseline = defaultValue(options.textBaseline, "bottom");
```

Comment:
Is this no longer relevant?

Reply:
I'm not sure why it was - the `measureText` function does not use the context's baseline anywhere. Maybe 9 years ago it worked differently? I didn't notice any changes when I removed this line.